projects/apps/Nomis.py

```diff
@@ -13,6 +13,10 @@
     metrics=[
         SmartContractInteraction(
             "Interaction", "EQBykWvdmoyFD2kB4BJbdrxRqyWKzSLDn-SgwNvznRfbsaKv"
+        ), 
+        SmartContractInteraction( 
+            "Nomis Boost", 
+            "UQBPIFKq_HLbqoksBqPGIMjFjTco2NH7o0ckV5Iocj_A22J2" 
```

Comment:
please use bounceable format for the address

Reply:
Done